apps.yaml

```
@@ -22,6 +22,16 @@ apps:
       name: "wildcard-hmcts-net"
     externalAuthenticationType: "passthru"
     hideApp: true
+  - name: "CFT Jenkins - Sandbox SonarQube"
+    externalUrl: "https://cft-sandbox-sonarqube-webhook.hmcts.net"
+    logoUrl: https://raw.githubusercontent.com/hmcts/azure-app-proxy/main/logos/jenkins.png
+    internalUrl: "https://sandbox-build.hmcts.net/sonarqube-webhook"
```

App proxy internal urls have to end with a / and i'm not sure if path based routing will work like how you've tried, I've not tested it, I would just pass the whole thing through

This must be your new ingress though with the path based route exposed:
```suggestion
    internalUrl: "https://cft-sandbox-sonarqube-webhook.hmcts.net"
```